.gitattributes

```diff
@@ -20,3 +20,11 @@ contrib/* linguist-vendored
 docs/media/**/*.png filter=lfs diff=lfs merge=lfs -text
 docs/media/**/*.gif filter=lfs diff=lfs merge=lfs -text
 docs/media/**/*.svg filter=lfs diff=lfs merge=lfs -text
+
+# TODO(rwcarlsen): Remove this after a while.
+framework/include/**/*.h merge=clang-format
+framework/src/**/*.C     merge=clang-format
+test/include/**/*.h      merge=clang-format
+test/src/**/*.C          merge=clang-format
+modules/**/include/**/*.h   merge=clang-format
+modules/**/src/**/*.C       merge=clang-format
```

Comment:
No - the user has to set it up in their git config and put the merge driver in their path for this to do anything.